onnxruntime/core/providers/js/operators/instance_norm.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/js/js_kernel.h"
+
+namespace onnxruntime {
+namespace js {
+
+class InstanceNorm : public JsKernel {
```

Comment:
we have a macro to replace the whole class definition: `JSEP_CLASS_IMPL_ATTRIBUTE_FLOAT_DEFAULT` (defined in js_kernel.h)

Reply:
The macro is not applicable in this case anymore because the class has defines two attributes, one float and the other int32_t.